### ISSUE TITLE
Columns in "our team"-section after updating

### ISSUE THREAD

**Issue by MickCgn4711** [Codeinwp/zerif-lite#536]
Hi,

I got a problem with the "our-team"-widget of Zerif Lite.
I updated the website an then the layout broke in this section.

The widgets are now vertically on the left side in one columns and not as they used to in the center and beside of each other. 

Is it a bug? Is there any way to change to the old layout?

It would be perfect if anyone had any idea how to solve this problem.

Best regards
Mick

**Comment by MickCgn4711** [comment]
All right! Problem solved! ;)

**Comment by MickCgn4711** [comment]
Now there still is the problem with the display on tablets.
I'm using Vantage Premium and the Siteorigin Page Builder.

**Comment by MickCgn4711** [comment]
Oops... there is another problem... 
can anyone tell, why there's a vertical stroke between the responsive menu an the search button?

![menu](https://cloud.githubusercontent.com/assets/22216947/23831652/44bd0736-0726-11e7-8786-3efda31983fc.PNG)

**Comment by rodica-andronache** [comment]
Hi Mick,

Thank you for contacting us. Can you please give me your site URL so I can see the issue?

Regards,
Rodica

**Comment by MickCgn4711** [comment]
Hi Rodica-Elena,

the URL is www.hoffmann-hsc.de

best regards
Michael

2017-03-13 11:12 GMT+01:00 Rodica-Elena Andronache <notifications@github.com

**Comment by rodica-andronache** [comment]
Hi Mick,

There seems to be a compatibility issue with the child theme you are using. Can you please try to add this code in the Additional CSS tab in Appearance -> Customize:
`.our-team .col-lg-3:nth-child(4n+1) {
     clear:none !important;
}`
And let me know if that helps.

Regards,
Rodica

**Comment by MickCgn4711** [comment]
Hi Rodica,

that was brilliant! It's all alright again now!
Thanks for your help!

I got one last question: Unfortunately in the footer-section suddenly there
are four rows now instead of three (the "copyright" on the right).
How is it possible to delete that?
I chose to put this on display:none so that it isn't shown. But I would
love to delete it to set the three columns centered in the footer-section.

Many greetings and best regards
Mick

2017-03-13 17:00 GMT+01:00 Rodica-Elena Andronache <notifications@github.com
